utoipa-gen/src/path/parameter.rs

```diff
@@ -238,6 +260,21 @@ impl FromStr for ParameterIn {
     }
 }
 
+impl Parse for ParameterIn {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
+        const EXPECTED_STYLE: &str = "unexpected in, expected one of: Path, Query, Header, Cookie";
+        let style = input.parse::<Ident>()?;
+
+        match &*style.to_string() {
+            "Path" => Ok(Self::Path),
+            "Query" => Ok(Self::Query),
+            "Header" => Ok(Self::Header),
+            "Cookie" => Ok(Self::Cookie),
+            _ => Err(Error::new(style.span(), EXPECTED_STYLE)),
```

Comment:
If we change the other location to CamelCase that will be a breaking change, but it's also easier to recognise it as being one of the enum variants. I agree with you that it should be consistent, which direction would you like to take it?

Reply:
Considering this library is already 1.0 I guess the aim is to reduce breaking changes where possible, perhaps I'll keep this as lower case?

Reply:
We can have it Pascal case. It will be breaking change but then it's going to be one. I consider the major release as a "how many mistakes have been done" and it can be increased according to the semver. But for sure it if it is possible to avoid breaking releases then it should be avoided as much as possible. But in this case it is fine for me. Then I can introduce other breaking changes as well with next release. So next release will then be `2.0.0`

Reply:
Okay sounds good I'll try to share the parsing then for this.

Reply:
@juhaku how does it look now with https://github.com/juhaku/utoipa/pull/150/commits/451ca013e8629769a54d41a6e10c2210b8b25fef ?

Reply:
Super that looks good